ratschlab_common/db/utils.py

```diff
@@ -0,0 +1,96 @@
+import attr
+import pgpasslib
+import records
+import sqlalchemy
+from records import RecordCollection
+from sqlalchemy.engine.reflection import Inspector
+
+@attr.s
+class PostgresDBParams(object):
+    user = attr.ib(default='postgres')
+    password = attr.ib(default=None, type=str)
```

Comment:
Thanks for the hint, I'm not super familiar with this library yet. I'll keep it the way it is though as here I don't see much added value in a parameter validation, since IMHO this is better handled by sqlalchemy/postgres client